### PR TITLE
Update git url for SortableJS

### PR DESCRIPTION
**Pull request by christiansk**
It seems the github repo for this library was moved out of the original developer's github account to its own project account.
- Old repo: https://github.com/RubaXa/Sortable
- New repo: https://github.com/SortableJS/Sortable
Also seems the latest version is v1.7.0.

### CODE REVIEW

File: ajax/libs/Sortable/package.json [hunk 1]
```diff
@@ -1,13 +1,12 @@
 {
   "name": "Sortable",
   "title": "Sortable",
-  "version": "1.6.0",
```
Comment (PeterDaveHello) [inline ajax/libs/Sortable/package.json]: Please do not manually touch the version info here.

Comment (christiansk) [reply ajax/libs/Sortable/package.json]: Sorry about that; still learning how this works.

**Comment by christiansk** [comment]
@PeterDaveHello I returned the version info to what it was previously. I understand the latest version will always be pulled from the project's git repom is that correct?
Is there anything else I should do to get this merged?
Thanks!

**Comment by christiansk** [comment]
Closing this since apparently it's not needed. I think due to the Github redirection to the new repository new releases are already being fetched from there.

**Pull request closed without merging**